### PR TITLE
chore: Add issue template for Refactoring tickets

### LINKED ISSUE #2980: 🚀 [Feature] Add issue template for Refactors
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T13:08:34Z" title="Tuesday, February 4th 2020, 2:08:34 pm +01:00">Feb 4, 2020</time>_
_Closed <time datetime="2020-02-04T19:02:51Z" title="Tuesday, February 4th 2020, 8:02:51 pm +01:00">Feb 4, 2020</time>_
---

### User Problem

I find myself needing to choose between `Bug` and `Feature`, when sometimes I want to create a `Refactor` ticket. 
### Implementation

We can add another template for `Refactor`

### PR DESCRIPTION
**Pull request by Tirokk**
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T13:32:16Z" title="Tuesday, February 4th 2020, 2:32:16 pm +01:00">Feb 4, 2020</time>_
_Merged <time datetime="2020-02-04T19:02:51Z" title="Tuesday, February 4th 2020, 8:02:51 pm +01:00">Feb 4, 2020</time>_
---

- We want to have an issue created for each PR merged in, sometimes we
  are just refactoring deprecated or legacy code and there should be an
option to create these tickets.

### Issues

- fixes #2980 

### Todo

- [X] None

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/refactor_tickets.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: 🔧 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🔧 [Refactor]
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the problem is.-->
```
Comment (Tirokk) [inline .github/ISSUE_TEMPLATE/refactor_tickets.md:9]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-04T13:35:06Z" title="Tuesday, February 4th 2020, 2:35:06 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,19 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the bug is.-->
```


```suggestion
<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the problem is.-->
```

File: .github/ISSUE_TEMPLATE/refactor_tickets.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+### Motive
+<!-- What is the purpose of this refactoring? If it's removing depcrecated code, please link to the deprecation notice. -->
+### Related issues
+<!-- Are there any related issues to link to? Please paste them below for reference. -->
+
```
Comment (Tirokk) [inline .github/ISSUE_TEMPLATE/refactor_tickets.md:15]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-04T13:35:20Z" title="Tuesday, February 4th 2020, 2:35:20 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,19 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the bug is.-->
+
+### Motive
+<!-- What is the purpose of this refactoring? If it's removing depcrecated code, please link to the deprecation notice. -->
+
+### Implementation
+<!-- Please, docment any ideas of how the code should be refactored. -->
```


```suggestion
<!-- Please, document any ideas of how the code should be refactored. -->
```

File: .github/ISSUE_TEMPLATE/refactor_tickets.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: 🔧 Refactor ticket
```
Comment (Tirokk) [inline .github/ISSUE_TEMPLATE/refactor_tickets.md:2]: > [alina-beck](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-04T14:22:32Z" title="Tuesday, February 4th 2020, 3:22:32 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
```


```suggestion
name: 🔧 Refactor ticket
```

may I suggest a different icon, to make it easier to distinguish between the issue types? :wink:

File: .github/ISSUE_TEMPLATE/refactor_tickets.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🔧 [Refactor]
```
Comment (Tirokk) [inline .github/ISSUE_TEMPLATE/refactor_tickets.md:5]: > [alina-beck](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-04T14:23:29Z" title="Tuesday, February 4th 2020, 3:23:29 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
```


```suggestion
title: 🔧 [Refactor] 
```

same here – feel free to choose another one, of course! it was just the first one that came to mind...

File: .github/ISSUE_TEMPLATE/refactor_tickets.md (hunk 2 above)
Comment (Tirokk) [inline .github/ISSUE_TEMPLATE/refactor_tickets.md:15]: > [alina-beck](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-04T14:25:18Z" title="Tuesday, February 4th 2020, 3:25:18 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
+---
+
+## :zap: Refactor ticket
+<!-- Describe your issue in detail. Include screenshots if needed. Give us as much information as possible. Use a clear and concise description of what the bug is.-->
+
+### Motive
+<!-- What is the purpose of this refactoring? If it's removing depcrecated code, please link to the deprecation notice. -->
+
```


Would a section `Related issues` be helpful, to point out where this came up? Or is it too much? I usually add it under `Additional context` when its relevant but know that not everyone does. :smile:

I'm ok with it either way.

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-02-04T15:25:15Z" title="Tuesday, February 4th 2020, 4:25:15 pm +01:00">Feb 4, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/4844/specs"></a> <b title="Passed">59</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/4844/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 5bbba483f8 |
| **Started** | Feb 4, 2020 4:15 PM |
| **Ended** | Feb 4, 2020 4:30 PM |
| **Duration** | 15:21 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Electron 78  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/4844/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

**Review by Mogge — APPROVED** [review]
I'm going to approve this so you don't have to wait another round of review until you can merge this. @mattwr18 just pull in these two changes.

**Review by Mogge — APPROVED** [review]
Thanks @mattwr18, this was long overdue! 💚

**Pull request closed without merging**